test/lib/enormous_table.hh

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include "mutation_reader.hh"
+
+namespace enormous_table {
+
+extern logging::logger elogger;
+
+class enormous_table_reader final : public flat_mutation_reader::impl {
```

Comment:
You put this code in lib/, do you think there's a chance it will ever be reused in other tests? If not, probably we don't need it in lib. But if you do, please add a comment explaining what this class does - that it reads into a virtual table with 4.5 billion rows.

Reply:
We probably won't use it any more. I'll include it within the test and add a comment